Application/EdFi.Ods.AdminApi/Infrastructure/Extensions/AdminModelExtensions.cs

```diff
@@ -23,4 +23,14 @@ public static class AdminModelExtensions
     {
         return application?.Profiles?.FirstOrDefault()?.ProfileName;
     }
+
+    public static int? ProfileId(this Application application)
+    {
+        return application?.Profiles?.FirstOrDefault()?.ProfileId;
```

Comment:
What if there is more than one Profile?

Reply:
For right now let's do this:

```json
"profiles": [
   { "profileId": 1 },
   { "profileId": 2 }
 ]
```

And then we can adjust if we get any feedback.